.github/workflows/main.yml

```diff
@@ -33,13 +32,20 @@ jobs:
       with:
         python-version: '3.7'
         architecture: 'x64'
-    - name: Install the dependencies
+    - name: Install the Python dependencies
       run: python -m pip install jupyterlab
-    - name: Install the extension
+    - name: Bootstrap the jlpm deps
+      run: jlpm
+    - name: Build all the extensions
+      run: |
+        jlpm build-ext
+        jlpm link-ext
+        jlpm build-jlab
```

Comment:
We might want to move this to a `build_all` job, but it's fine if we do it in a separate PR.

Reply:
Have created a separated build_all.yml in `.github/workflows` - I am not used to GitHub CI so I will push and see how it behaves